dashboard/app/helpers/levels_helper.rb

```diff
@@ -125,6 +125,7 @@ def app_options
       autoplay_video: select_and_track_autoplay_video,
       callouts: callouts
     )
+    view_options(is_external_project_level: true) if @level.pixelation?
```

Comment:
What does it mean to be external_project_level? Not sure where the best place for such a comment is, but seems like it would be nice to have one.